fmatch/test_fmatch.py

```diff
@@ -2,35 +2,47 @@
 test_fmatch
 """
 
+from datetime import datetime
 import sys
+import warnings
 # pylint: disable=import-error
 import pandas as pd
 
 # pylint: disable=import-error
 from matcher import Matcher
 
-match = Matcher(index="perf_scale_ci")
-res=match.get_metadata_by_uuid("b4afc724-f175-44d1-81ff-a8255fea034f",'perf_scale_ci')
+warnings.filterwarnings("ignore", message="Unverified HTTPS request.*")
+warnings.filterwarnings(
+    "ignore", category=UserWarning, message=".*Connecting to.*verify_certs=False.*"
+)
+
+match = Matcher(index="perf_scale_ci*", verify_certs=False)
+res=match.get_metadata_by_uuid("b4afc724-f175-44d1-81ff-a8255fea034f",'perf_scale_ci*')
 
 meta = {}
 meta["masterNodesType"] = "m6a.xlarge"
 meta["workerNodesType"] = "m6a.xlarge"
 meta["platform"] = "AWS"
 meta["masterNodesCount"] = 3
-meta["workerNodesCount"] = 24
+meta["workerNodesCount"] = 6
 meta["jobStatus"] = "success"
-meta["ocpVersion"] = "4.15"
+meta["ocpVersion"] = "4.17"
 meta["networkType"] = "OVNKubernetes"
 meta["benchmark.keyword"] = "cluster-density-v2"
 # meta['encrypted'] = "true"
 # meta['ipsec'] = "false"
 # meta['fips'] = "false"
 
 uuids = match.get_uuid_by_metadata(meta)
+print("All uuids",len(uuids))
```

Comment:
Are these just debugs?

Reply:
yep this is just a utility for functionality testing.